.pre-commit-config.yaml

```
@@ -39,7 +39,6 @@ repos:
     hooks:
       - id: remove-tabs
         types: [python]
```

Let's keep the exclude for now, we can remove it later if we backport https://github.com/python/cpython/pull/137622.

```suggestion
        types: [python]
        exclude: ^Tools/c-analyzer/cpython/_parser.py
```